hydra_genetics/utils/io/hotspot_report.py

```diff
@@ -191,8 +191,10 @@ def add_columns(data, var, hotspot, columns, annotation_extractor, depth, levels
             if columns["columns"][c]['from'] == "vep":
                 try:
                     data[c] = annotation_extractor(var,  columns["columns"][c]['field'])
+                    if data[c] is None:
+                        data[c] = '-'
                 except AttributeError:
-                    data[c] = "-"
+                    data[c] = '-'
```

Comment:
There is a lot of double and single quotes everywhere. I would stick to one or the other :wink: